### PR TITLE
Add quizzes for leanpub and coursera

### PR DESCRIPTION
**Pull request by avahoffman**
Appreciate your comments, @carriewright11 !

### CODE REVIEW

File: quizzes/quiz_2.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+o) Access, distribution, reuse considerations - describe how you have carefully considered any reasons that might limit sharing
+o) Oversight - describe who will manage compliance of the DMS plan
+
+{choose-answers: 4}
+? What factors influence your data storage needs?
+
+C) The type of data and the number of samples
+m) The number of researchers working on the project
+m) The location of the data storage facility
+m) The funding agency's requirements for data storage
```
Comment (carriewright11) [inline quizzes/quiz_2.md:26]: m) The funding agency's requirements for data storage ...I think this answer could be correct too actually- like that your agency requires that you share all raw data for example

File: quizzes/quiz_2.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+C) The choice of repository is not always obvious, and sometimes a little research into an appropriate one is needed
+o) The repository is always listed in the funding announcement (FOA) / request for applications (RFA)
+o) Only one appropriate repository exists for each data type
+o) Generalist repositories are for storing data collected from humans
+o) NIH discourages repositories that generate digital object identifiers (DOIs)
+o) The repository is the place where you upload your DMS plan
+
+{choose-answers: 4}
+? What costs related to data management can be included in the budget for a research project?
```
Comment (carriewright11) [inline quizzes/quiz_2.md:39]: could ask a question about if is budgeted as direct costs or some other incorrect options :P

File: quizzes/quiz_1.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+m) Results used to publish papers
+
+{choose-answers: 4}
+? Which of the following is FALSE regarding the grant review process and your DMS Plan??
+
+m) Reviewers will not have access to your DMS plan.
+m) Reviewers will see your budget which will include some descriptions of how money will be spent to manage and share data.
+m) DMS plan should not influence your grant score.
+C) The Program Officer will write the DMS plan and will work with the PI to address any concerns.
+C) The Program Officer will score the plan and add the value to your grant score.
```
Comment (carriewright11) [inline quizzes/quiz_1.md:45]: C) The Program Officer will score the plan and add the value to your grant score.

I think this one is an incorrect answer

File: quizzes/quiz_2.md (hunk 1 above)
Comment (avahoffman) [reply quizzes/quiz_2.md:26]: Good point!

**Review by carriewright11 — APPROVED** [review]
Looks great! - just some small comments

File: quizzes/quiz_2.md (hunk 2 above)
Comment (avahoffman) [reply quizzes/quiz_2.md:39]: ok!

**Pull request merged**